packages/api/src/external/ehr/elation/command/process-patients-from-appointments.ts

```diff
@@ -118,12 +122,22 @@ async function syncPatient({
   elationPracticeId,
   elationPatientId,
 }: Omit<SyncElationPatientIntoMetriportParams, "api" | "triggerDq">): Promise<void> {
+  const cxMapping = await getCxMappingOrFail({
+    externalId: elationPracticeId,
+    source: EhrSources.elation,
+  });
+  const secondaryMappings = cxMapping.secondaryMappings as ElationSecondaryMappings;
```

Comment:
ditto: specific function per EHR, and fine to do that on a follow-up PR given the deadline